cropduster/templatetags/cropduster_tags.py

```diff
@@ -73,3 +73,19 @@ def get_crop(image, crop_name, size=None, attribution=None, exact_size=False):
             "caption": image.related_object.caption,
         })
     return data
+
+
+@register.assignment_tag
+def get_thumbs(image):
+    def _serialize_thumb(thumb):
+        return {
+            "url": Image.get_file_for_size(image, thumb.name).url,
```

Comment:
Here's the source for `get_image_url`:

```python
    def get_image_url(self, size_name='original', tmp=False):
        converted = Image.get_file_for_size(self.image, size_name, tmp=tmp)
        return getattr(converted, 'url', None) or u''
```

It's doing the same thing as you are, but with a `getattr` that falls back to empty string. (I don't know if that would ever make a real world difference, but it's one less thing to blow up.) ISTM it's cleaner to use the method that gives you the url you want instead of doing it yourself and maybe having differences in how your code is implemented.